test_conformance/images/kernel_image_methods/test_1D_buffer.cpp

```diff
@@ -262,8 +262,7 @@ int test_get_image_info_1D_buffer(cl_device_id device, cl_context context,
                     imageInfo.rowPitch += extraWidth;
                 } while ((imageInfo.rowPitch % pixelSize) != 0);
 
-                size = (cl_ulong)imageInfo.rowPitch * (cl_ulong)imageInfo.height
```

Comment:
imageInfo.height is initialized to 0 earlier, so the loop exits right away.  The loop has not been doing anything useful.  Change the calculation to use an assumed height of 1.